source/multiple-mongosyncs.txt

```diff
@@ -133,6 +127,20 @@ To configure multiple ``mongosync`` instances:
            --cluster1 "mongodb://user:password@cluster1host:27500" \
            --id shard02 --port 27602
 
+      The connection strings for the :option:`--cluster0` and 
+      :option`--cluster1` options should point to ``mongos`` instances.
```

Comment:
[rendering issue]

Looks like there is a colon missing between "option" and the backtick.

Reply:
Fixed